pypsexec/client.py

```diff
@@ -100,6 +110,9 @@ def __init__(self, server, username=None, password=None, port=445,
                  % self._unique_id)
         self._service = Service(self.service_name, self.session)
 
+        for attribute,value in kargs.items():
+            setattr(self, attribute, value)
```

Comment:
to edit attributes.
_________________________________
`Client(current_host = 'localhost')`
>can modify self.current_host = 'localhost'